pkg/application/inject/fuse/injector.go

```diff
@@ -372,17 +320,15 @@ func (s *Injector) InjectUnstructured(in *unstructuredtype.Unstructured, runtime
 	return nil, fmt.Errorf("not implemented yet")
 }
 
-func (s *Injector) shouldInject(pod common.FluidObject, namespacedName types.NamespacedName) (should bool, err error) {
+func (s *Injector) shouldInject(pod common.FluidObject, log logr.Logger) (should bool, err error) {
```

Comment:
How about handling logger as an attribute of injector? We can avoid pass the parameter in function calling.

Reply:
Done. For consistency, some functions like `appendVolumes` and `randomizeNewVolumeName` are modified as a member of `Injector`